### DISCUSSION TITLE
Is it possible to visualize higher-dimensional polyhedra of ambient dimension <= 3  ?

### DISCUSSION THREAD

**Discussion by wittenator** [oscar-system/Oscar.jl#1452] [Q&A]
I am currently trying to visualize a permutahedron with dimensionality = 4 in Oscar. I created a polytope via ```Polymake.polytope.Polytope(POINTS=vertices)``` and a polyhedron via ```convex_hull```. Now I am interested in visualizing the resulting polytope.  I tried using the ```visualize``` method, but it fails for a dim > 3. The corresponding Polymake function ```Polymake.visual(...)``` on the other hand mangles the points, but visualizes something (I assume due to a different interpretation of the coordinate system?). Is it intended that there is a difference in functionality?

**Comment by micjoswig** [comment]
Sorry that I am seeing this only now.

The standard way of visualizing 4-polytopes is via Schlegel diagrams.  This is what you get from, e.g., `visualize(cube(4))` ; e.g., see Chapter 5 of [Zie95].  There is no standard way of visualizing polytopes in higher dimension; [here are some ideas](http://dx.doi.org/10.1007/s00791-009-0127-3).
